.azure-pipelines/client.yml

```diff
@@ -95,10 +95,28 @@ jobs:
           publishJUnitResults: false
           goals: 'site:site site:stage'
 
+      - script: |
+          git clone https://github.com/JonathanGiles/DependencyChecker.git
```

Comment:
@bsiegel @mitchdenny should we be publishing this package instead of cloning the repo during our builds?

Reply:
I think in general we should download tools that are published either to GH releases, or public registries (depending on what the tools are/do).

Reply:
I asked @JonathanGiles this exact question, his opinion at the time was that it was best to pull from source.

Reply:
I do think we will want to consume this via a published package at some point. We have the azure-sdk-tools repo specifically for produce such tool packages.

Reply:
I filed https://github.com/Azure/azure-sdk-tools/issues/44 as a follow-up on this.